docs/sources/panels/configure-panel-options/index.md

```diff
@@ -0,0 +1,140 @@
+
+1. [Edit a panel](#edit-a-panel).
+
+1. In the panel display options pane, locate the **Panel options** section.
+
+1. Enter a **Title**.
+
+   Text entered in this field appears at the top of your panel in the panel editor and in the dashboard.
+
+1. Write a description of the panel and the data you are displaying.
+
+   Text entered in this field appears in a tooltip in the upper-left corner of the panel.
+
+   You can use [variables you have defined]({{< relref "../../variables/" >}}) in the **Title** and **Description** field, but not [global variables]({{< relref "../../variables/variable-types/global-variables/" >}}).
+
+   ![](/static/img/docs/panels/panel-options-8-0.png)
+
+## View a panel JSON model
+
+Explore and export panel, panel data, and data frame JSON models.
+
+1. Open the dashboard that contains the panel.
+
+1. Click in the panel title and select **Inspect > Panel JSON**.
+
+1. In the **Select source** field, select one of the following options:
+
+   - **Panel JSON:** Displays a JSON object representing the panel.
+   - **Panel data:** Displays a JSON object representing the data that was passed to the panel.
+   - **DataFrame structure:** Displays the raw result set with transformations, field configurations, and override configurations applied.
+
+1. To explore the JSON, click `>` to expand or collapse portions of the JSON model.
+
+## Configure repeating rows or panels
+
+You can configure Grafana to dynamically add panels or rows to a dashboard. A dynamic panel (or row) is a panel that the system creates based on the value of a variable. Variables dynamically change your queries across all panels in a dashboard.
+
+> **Note:** Repeating panels require variables to have one or more items selected; you cannot repeat a panel zero times to hide it.
+
+Refer to the following links for examples of repeating panels and rows:
+
+- [Prometheus dashboard with repeating panels](https://play.grafana.org/d/000000036/prometheus-repeat)
+- [Dashboard with repeating rows](https://play.grafana.org/d/000000153/repeat-rows)
+
+The examples show that you can also repeat rows if you have variables set with `Multi-value` or
+`Include all values` selection option.
+
+### Configure repeating rows
+
+Complete the following steps to add repeating rows to a dashboard.
+
+**Before you begin:**
+
+- Ensure that the query includes a multi-value variable.
+
+**To configure repeating rows:**
+
+1. On the dashboard home page, click **Add panel**.
+
+1. On the **Add a panel** dialog box, click **Add a new row**.
+
+1. Hover over the row title and click the cog icon.
+
+1. On the **Row Options** dialog box, add a title and select the variable for which you want to add repeating rows.
+
+> **Note:** To provide context to dashboard users, add the variable to the row title.
```

Comment:
Configure repeating rows feels like it does not belong to this topic, but more a "build and organize a dashboard" topic, like this one: 
http://localhost:3002/docs/grafana/next/dashboards/add-organize-panels/